src/index.html

```diff
@@ -1,14 +1,53 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css"
+    href="style.css">
+    <link rel="stylesheet" type="text/css"
+    href="normalize.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a class="nav-logo" href="#">
+          <img src="images/logo.png" alt="moyo_img">
+        </a>
+        <nav>
+          <ul class="nav-list">
+            <li>
```

Comment:
and then add margins to this parent element, because now it looks weird
![image](https://github.com/mate-academy/layout_moyo-header/assets/55200530/1291a16a-e8cc-4ecc-8f9f-b4c44fb58bb3)